README.md

```diff
@@ -59,18 +59,13 @@ solutions. If you cannot find your specific problem then please create a
 
 Adding a new component
 ---
-#### Install our [yeoman component generator](https://github.com/holidayextras/generator-ui-component)
```

Comment:
Why are we removing this bit? I assume we are assuming everybody has it installed?